p2p/src/config.rs

```diff
@@ -63,6 +63,9 @@ pub struct Config<C: Crypto> {
     /// Duration after which to close the connection if a message cannot be written.
     pub write_timeout: Duration,
 
+    /// Whether or not to disable Nagle's algorithm
```

Comment:
nit: can you add a few more words here around "that means sending data as soon as possible". end line in a period like others.

Reply:
Would also be open to also adding (if they have some rationale somewhere):

```
///
/// By default, this is disabled in Golang because.....
```

Reply:
Just added some explanation about it. And on why it is disabled in Golang, I only have is the comment below from the person who wrote the dialer code on HN. I think the explanation I just added is sufficient.

`When I wrote the Go implementation of net.Dial, which I expected to be used for RPC-based systems, it seemed like a no-brainer to set TCP_NODELAY by default.`